recipes/imagehash/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.1" %}
+
+package:
+  name: imagehash
+  version: {{ version }}
+
+source:
+  fn: imagehash-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/0c/1c/b6f539cc08cf8cfb12cb59a6883ee7f1cc0937cf3709247c2a15c8a1e665/ImageHash-{{ version }}.tar.gz
```

Comment:
Can you change to the more readable `https://pypi.io/packages/source/I/ImageHash/ImageHash-{{ version }}.tar.gz`?